### ISSUE TITLE
Adjust the default `--testMatch` parameter passed to Jest to better reflect the internal patterns of Jest itself

### ISSUE THREAD

**Issue by Xiltyn** [callstack/reassure#363]
**Describe the improvement**
Consider updating the default testMatch to better reflect
default patterns used in Jest and allow users to also place their
performance tests into specific /__perf__/ directory without the need
of adding the *.perf-test. prefix 

```
[ **/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)" ]
```

**Scope of improvement**
Update the `measure` command in `reasure-cli` package 

from 
```
'<rootDir>/**/*.perf-test.[jt]s?(x)'
```
to
```
[ "<rootDir>/**/__perf__/**/*.[jt]s?(x)", "<rootDir>/**/*.perf-test.[jt]s?(x)" ]
```

**Suggested implementation steps**
One quick PR should do, additionally, we need to make sure there we're properly handling testMatch in any other place since it would now be an array as per the [Jest docs](https://jestjs.io/docs/configuration#testmatch-arraystring)

**Comment by thymikee** [comment]
I'm good with that

**Comment by thymikee** [comment]
It should be fine to drop the `<rootDir>`, considering that Jest doesn't use it by default for matching these globs

**Comment by mdjastrzebski** [comment]
Resolved in #338
